.vscode/settings.json

```diff
@@ -1,17 +1,4 @@
 {
-  "eslint.autoFixOnSave": true,
```

Comment:
This was automatically converted by the eslint plugin in my code editor. I checked to make sure that autosave still functioned properly after these changes were made

Reply:
Yup, looks like this comes with a vscode upgrade which deprecates the `eslint.autoFixOnSave` option https://code.visualstudio.com/updates/v1_41#_eslint

One change I did push up here is to move `editor.codeActionsOnSave` so that the linter rules come before the formatting rules